src/transformers/image_utils.py

```diff
@@ -245,25 +276,49 @@ def resize(self, image, size, resample=PIL.Image.BILINEAR, default_to_square=Tru
 
                 size = (new_short, new_long) if width <= height else (new_long, new_short)
 
-        return image.resize(size, resample=resample)
+        image.resize(size, resample=resample)
+
+        # Convert back to original image format
+        if image_format in ["np", "pt"]:
+            image = self.to_numpy_array(image)
+
+            if image_format == "pt":
+                import torch
+                image = torch.from_numpy(image)
+
+        return image
```

Comment:
I don't think we want to do this at all as it will slow down the preprocessing pipeline a lot. The conversion back will be done according to the `return_tensors` argument at the end of the feature extractor call.

Reply:
Thanks, I reverted the resize method back to the previous version.